app/templates/main.html

```diff
@@ -1,7 +1,18 @@
 <section>
 	<div>
+		<form>
+			<label>Add a Room!<br>
+			<input type="text" name="add_room" ng-model="roomName" placeholder="Room Name"></label>
+			<input type="button" value="Add Room!" ng-click="main.addRoom(roomName)">
```

Comment:
You don't need `main.` here if you update the controller to set the `addRoom` method on `$scope`.